package.json

```diff
@@ -6,7 +6,7 @@
     "examples/js-*",
     "src/messaging/js/*",
     "src/shell/js/*",
-    "Tryouts/Plugins/ApplicationPlugins/*"
+    "prototypes/*"
```

Comment:
I just noticed on the console, that the project doesn't seem to be picked up under the new location.
Could you change this to `prototypes/process-explorer/js` ?
If it's still not picked up, we can merge ad I'm looking at it locally.